### DISCUSSION TITLE
Server exited with error 134. Check whether OS is supported.

### DISCUSSION THREAD

**Discussion by senniraf** [UPPAALModelChecker/UPPAAL-Meta#111] [Q&A] [answered]
Hi, 

I'm Raffael a Student from the University of Konstanz. I have a similiar problem to #109. When I run uppaal on my Manjaro Desktop via the `./uppal` script, I get the following error message:
```
Server exited with error 134. Check whether OS is supported.

Java version: 18.0.1.1
Java vendor: N/A
JVM version: 18.0.1.1+2
JVM vendor: Oracle Corporation
JVM name: OpenJDK 64-Bit Server VM
JRE version: 18
JRE vendor: Oracle Corporation
JRE name: Java Platform API Specification
OS name: Linux
OS arch: amd64
OS version: 5.15.41-1-MANJARO
```
Uppaal is starting but I cannot simulate or verify and there is a "License installation" dialog popping up.

When I first start the socketserver via `bin-Linux/socketserver -dm` and Uppaal afterwards via `java -jar uppaal.jar` I get the following error message from Uppaal: 
```
Unknown protocol version: error

Java version: 18.0.1.1
Java vendor: N/A
JVM version: 18.0.1.1+2
JVM vendor: Oracle Corporation
JVM name: OpenJDK 64-Bit Server VM
JRE version: 18
JRE vendor: Oracle Corporation
JRE name: Java Platform API Specification
OS name: Linux
OS arch: amd64
OS version: 5.15.41-1-MANJARO
```

And the following output from the socketserver:
```
UPPAAL Socket Server
Copyright (c) 1995 - 2000, Aalborg University and Uppsala University
All rights reserved.
Listening on port 2350
Connection from 127.0.0.1 accepted
Caught signal 17, closing proxy.
5195: ERROR /usr/bin/env: /home/raffael/Tools/uppaal64-4.1.26-1/bin-Linux/libc.so.6: version `GLIBC_2.34' not found (required by /usr/bin/env)
Connection from 127.0.0.1 closed
```

When I try the work-around from #109 `socat TCP-LISTEN:2350,reuseaddr,fork EXEC:"./server"` I get the following output:
```
Internet connection is required for activation.
2022/05/24 11:13:17 socat[5377] E waitpid(): child 5378 exited on signal 6
```

And the following error message from Uppaal:
```
Server closed connection.

Java version: 18.0.1.1
Java vendor: N/A
JVM version: 18.0.1.1+2
JVM vendor: Oracle Corporation
JVM name: OpenJDK 64-Bit Server VM
JRE version: 18
JRE vendor: Oracle Corporation
JRE name: Java Platform API Specification
OS name: Linux
OS arch: amd64
OS version: 5.15.41-1-MANJARO
```
Is there maybe another work-around?

Best regards
Raffael

**Comment by mikucionisaau** [comment]
@senniraf did you download the academic or commercial version of Uppaal?
Please paste what does `verifyta -v` produce

**Comment by RasmusRendal** [comment, marked as answer]
A similar issue seems to be posted here, with a workaround: https://github.com/UPPAALModelChecker/UPPAAL-Meta/issues/75

**Reply by senniraf** [reply]
Thanks. That solved the issue
